src/overlays/actors/ovl_Arrow_Ice/z_arrow_ice.c

```diff
@@ -9,13 +11,16 @@ void ArrowIce_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void ArrowIce_Update(Actor* thisx, GlobalContext* globalCtx);
 void ArrowIce_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-void func_809224DC(ArrowIce* this, GlobalContext* globalCtx);
-void func_80922628(ArrowIce* this, GlobalContext* globalCtx);
-void func_809227F4(ArrowIce* this, GlobalContext* globalCtx);
+void ArrowIce_Charge(ArrowIce* this, GlobalContext* globalCtx);
+void ArrowIce_Hit(ArrowIce* this, GlobalContext* globalCtx);
+void ArrowIce_Fly(ArrowIce* this, GlobalContext* globalCtx);
 
 void ArrowIce_SetupAction(ArrowIce* this, ArrowIceActionFunc actionFunc);
```

Comment:
```suggestion
void ArrowIce_Fly(ArrowIce* this, GlobalContext* globalCtx);
void ArrowIce_SetupAction(ArrowIce* this, ArrowIceActionFunc actionFunc);
```

Reply:
I'll do you one better and just remove the `SetupAction` forward declaration, since it's not needed.